content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
 {{< code/copyable >}}
 {
   "require": {
-    "php": ">=7.1"
+    "php": ">=8.0"
   }
 }
 {{< /code/copyable >}}
 
 If your app has a `composer.lock` file, the buildpack will use
 the php version defined there.
 
+#### Deprecated: Using buildpack.yml
+
+Specifying the PHP version through `buildpack.yml` configuration will be
+deprecated in PHP language family buildpack v1.0.0.  To migrate from using
+`buildpack.yml` please set the `$BP_PHP_VERSION` environment variable.
+
+### Configure the PHP library directory
+
+Users can override the default libray code directory of `lib` for PHP by
+setting the `BP_PHP_LIB_DIR` environment variable at build-time.
+{{< code/copyable >}}
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_LIB_DIR=some-lib-directory
+{{< /code/copyable >}}
+
+### Configure PHP with a custom `.ini` file
+If you'd like to configure custom `.ini` files in addition to the default
+`php.ini` provided by the [PHP Dist buildpack][bp/php-dist], users can provide
+their own configuration file in the application source directory under a
+directory named `.php.ini.d/`. Check out the reference docs about the
+[`PHP_INI_SCAN_DIR`]({{< ref "docs/reference/php-reference#php_ini_scan_dir"
+>}}) for more information about defaults.
+
+### Use Extensions
+There are two ways to enable extensions when using the Paketo PHP Buildpack.
+The only extensions available for usage at this time are the ones that come
+with the distribution of PHP.
```

Comment:
Are these enumerated somewhere? If so, a link here would be helpful.

Reply:
I intentionally left this off. It is ONLY defined in the v2 buildpacks but I intend to fully rework this soon / make it available in release notes on the v3, so I felt it was easier to leave off for now.

Reply:
Makes sense to me! Is there an issue tracking this improvement to the PHP release notes?

Reply:
https://github.com/paketo-buildpacks/php-dist/issues/381